PWGHF/TableProducer/candidateSelectorD0.cxx

```diff
@@ -158,7 +158,7 @@ struct HfCandidateSelectorD0 {
       return false;
     }
     // candidate DCA
-    // if (candidate.chi2PCA() > cuts[pTBin][1]) return false;
+    if (candidate.chi2PCA() > cuts[pTBin][1]) return false;
```

Comment:
beware that in this way you are applying a cut on the `chi2PCA`, while instead you want to apply a cut on candidate DCA xy, which can be got with `impactParameterXY` (see here: https://github.com/AliceO2Group/O2Physics/blob/master/PWGHF/DataModel/CandidateReconstructionTables.h#L624). Please fix it

Reply:
Hi, @mfaggin! Thanks for the reminder, I forgot to save the uncomment.
Should I use the variable below?
```suggestion
    if (candidate.ImpactParameterXY() > cuts[pTBin][1]) return false;
```

Reply:
yes correct

Reply:
But Ninja has failed, so should I add the DECLARE_SOA_DYNAMIC_COLUMN like below:
https://github.com/AliceO2Group/O2Physics/pull/7555/commits/573dec41b88f258fe5d220ed14ffd69e3a3deb50